app/root.tsx

```diff
@@ -1,19 +1,40 @@
-import type { LinksFunction, LoaderArgs } from '@remix-run/node';
+import {
+  Links,
+  LiveReload,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+  isRouteErrorResponse,
+  useRouteError,
+} from '@remix-run/react';
+import type { LinksFunction, LoaderFunctionArgs, MetaFunction } from '@remix-run/node';
 import { json } from '@remix-run/node';
 import stylesheet from './styles/tailwind.css';
 import { TopNav } from '~/components/layout/top-nav';
-import { Links, LiveReload, Meta, Outlet, Scripts, ScrollRestoration } from '@remix-run/react';
 import Footer from './components/layout/footer';
+import { ErrorMessage } from '~/components/ui/error-message';
 import { getCurrentUser } from '~/modules/session/session.server';
 import { db } from './modules/database/db.server';
 
+export const meta: MetaFunction = ({ error }: any) => {
```

Comment:
:any can be removed as it's type is inferred by the MetaFunction type that's already there

Reply:
This is a workaround) Did it but TypeScript's complaining with "Property 'error' does not exist on type 'ServerRuntimeMetaArgs<unknown, Record<string, unknown>>'.ts(2339)". 
It also worked when I used the error argument without destructuring. Not sure if it's a good idea...